internal/server/v2/resolve/golden/resolve_description_test.go

```diff
@@ -97,6 +97,16 @@ func TestResolveDescription(t *testing.T) {
 				"<-description->dcid",
 				"harvard.json",
 			},
+			{
+				[]string{
+					"California, USA",
+					"Mountain View, US",
+					"Sunnyvale CA",
+					"ME",
```

Comment:
I guess "CA US" should work too?

Reply:
Yes it resolves to "geoId/06" -- added to resolve_description_test.go.